infer/src/base/DB.ml

```diff
@@ -23,9 +25,10 @@ let append_crc_cutoff ?(key = "") name =
   in
   let crc_str =
     let name_for_crc = name ^ key in
-    Utils.string_crc_hex32 name_for_crc
+    String.sub (Utils.string_crc_hex32 name_for_crc) ~pos:0 ~len:16
```

Comment:
Windows _paths_ are limited to 260 characters, unless you use the long-filenames API (which comes with its own set of drawbacks). Since the path prefix is not directly under Infer's control, anything that makes the total length small is welcome.